templates/istio.yaml

```diff
@@ -55,4 +55,12 @@ spec:
             topologyKey: node-role.kubernetes.io/control-plane
     istio_cni:
       enabled: true
+---
+apiVersion: v1
+kind: Secret
+data:
```

Comment:
Could you use stringData for this to make our future lives easier?

Reply:
Good call